langserve/server.py

```diff
@@ -35,22 +37,22 @@
 from langsmith.utils import tracing_is_enabled
 from typing_extensions import Annotated
 
+try:
+    from pydantic.v1 import BaseModel, Field, ValidationError, create_model
+except ImportError:
```

Comment:
nit: why use the import error here rather than the `if PYDANTIC_MAJOR_VERSION == 2:` syntax used elsewhere?

Reply:
I think it makes sense to update seprately to remove the conditional logic completely like we have in langchain and just import from a v1 namespace